aspnetcore/security/authentication/claims.md

```diff
@@ -104,7 +104,7 @@ public void ConfigureServices(IServiceCollection services)
 
 ## Claims namespaces, default namespaces
 
-ASP.NET Core adds default namespaces to some known claims which might not be required in your app. You can disable this namespace adding if you would like to use the claims exactly like the Open ID Connect server created. 
+ASP.NET Core adds default namespaces to some known claims which might not be required in your application. You can disable this namespace adding if you would like to use the claims exactly like the Open ID Connect server created. 
```

Comment:
Routinely, we use "app" everywhere, so that bit is ok. However, the "you/your" bits and some of the other phrasing could use a touch, and there's a missing comma for the non-essential clause.

```suggestion
ASP.NET Core adds default namespaces to some known claims, which might not be required in the app. Optionally, disable these added namespaces and use the exact claims that the OpenID Connect server created.
```

... something along those lines.

Reply:
thanks for the review, updated with you improvements